exaudfclient/base/javacontainer/BUILD

```diff
@@ -50,26 +66,49 @@ genrule(
+            find "$$TMPDIR"/java_src/com/exasol/swig/ -name *.java -type f -exec chmod 644 {} \\;
+            cp -r -L $(location //:filter_swig_code.py) $(location :exascript_java_tmp.cc) "$$TMPDIR"
+            (cd "$$TMPDIR"
             python3 filter_swig_code.py "exascript_java.cc" "exascript_java_tmp.cc")
-            cp $$TMPDIR/exascript_java.cc "$(location exascript_java.cc)"
-            cp -r $$TMPDIR/java_src/com/exasol/swig/ConnectionInformationWrapper.java "$(location ConnectionInformationWrapper.java)"
-            cp -r $$TMPDIR/java_src/com/exasol/swig/exascript_java.java "$(location exascript_java.java)"
-            cp -r $$TMPDIR/java_src/com/exasol/swig/exascript_javaJNI.java "$(location exascript_javaJNI.java)"
-            cp -r $$TMPDIR/java_src/com/exasol/swig/ExportSpecificationWrapper.java "$(location ExportSpecificationWrapper.java)"
-            cp -r $$TMPDIR/java_src/com/exasol/swig/ImportSpecificationWrapper.java "$(location ImportSpecificationWrapper.java)"
-            cp -r $$TMPDIR/java_src/com/exasol/swig/Metadata.java "$(location Metadata.java)"
-            cp -r $$TMPDIR/java_src/com/exasol/swig/ResultHandler.java "$(location ResultHandler.java)"
-            cp -r $$TMPDIR/java_src/com/exasol/swig/SWIGTYPE_p_ExecutionGraph__ExportSpecification.java "$(location SWIGTYPE_p_ExecutionGraph__ExportSpecification.java)"
-            cp -r $$TMPDIR/java_src/com/exasol/swig/SWIGTYPE_p_ExecutionGraph__ImportSpecification.java "$(location SWIGTYPE_p_ExecutionGraph__ImportSpecification.java)"
-            cp -r $$TMPDIR/java_src/com/exasol/swig/SWIGVM_datatype_e.java "$(location SWIGVM_datatype_e.java)"
-            cp -r $$TMPDIR/java_src/com/exasol/swig/SWIGVM_itertype_e.java "$(location SWIGVM_itertype_e.java)"
-            cp -r $$TMPDIR/java_src/com/exasol/swig/TableIterator.java "$(location TableIterator.java)"
-            rm -rf $$TMPDIR
+            cp "$$TMPDIR"/exascript_java.cc $(location :exascript_java.cc)
+            cp "$$TMPDIR"/java_src/com/exasol/swig/TableIterator.java $(location :filtered/com/exasol/swig/TableIterator.java)
+            cp "$$TMPDIR"/java_src/com/exasol/swig/ResultHandler.java $(location :filtered/com/exasol/swig/ResultHandler.java)
+            cp "$$TMPDIR"/java_src/com/exasol/swig/exascript_javaJNI.java $(location :filtered/com/exasol/swig/exascript_javaJNI.java)
+            rm -rf "$$TMPDIR"
             """,
     outs = [
             "exascript_java.cc",
+            "filtered/com/exasol/swig/TableIterator.java",
+            "filtered/com/exasol/swig/ResultHandler.java",
+            "filtered/com/exasol/swig/exascript_javaJNI.java"
+            ],
+    srcs = [":exascript_java_tmp.cc", ":orig/com/exasol/swig/TableIterator.java",
+            ":orig/com/exasol/swig/ResultHandler.java", ":orig/com/exasol/swig/exascript_javaJNI.java"],
+    tools = ["//:filter_swig_code.py"]
+)
+
+
+genrule(
+    name = "finish_java_package",
```

Comment:
Do we really need this, what happens when we put all the inputs of this rule into the jar rule?

Reply:
Then the directories of the files does not match

Reply:
Done